### ISSUE TITLE
Queue

### ISSUE THREAD

**Issue by harddiegg** [arybczak/ncmpcpp#329]
Does the player have a function to add a song to the queue?
So that at the end of the current song the next one in the list is not played, but a song from the queue is played?

**Comment by GrimKriegor** [comment]
Indeed, I have missed this feature ever since I migrated from cmus.

**Comment by Solarunit** [comment]
The queue is essential for any music player. I would like to have an ability to add the same song(s) into queue unlimited times. There could be some numeral symbols for tracks added to the queue. The queue could have a separate window.
